st2client/st2client/models/core.py

```diff
@@ -337,6 +339,20 @@ def __init__(self, resource, endpoint, cacert=None, debug=False):
         self.debug = debug
         self.client = httpclient.HTTPClient(root=endpoint, cacert=cacert, debug=debug)
 
+    @add_auth_token_to_kwargs_from_env
+    def match(self, command, **kwargs):
+        url = '/%s/match' % self.resource.get_url_path_name()
+        query_str = urlencode({'command': command})
+        response = self.client.post_raw(url, query_str, **kwargs)
```

Comment:
Hm, this is sent as query string, right?

If so, requests library should encode it correctly as long as the requests.post method is called with the params=params argument.